tornado/netutil.py

```diff
@@ -47,7 +47,7 @@
 "foo".encode("idna")
 
 # For undiagnosed reasons, 'latin1' codec may also need to be preloaded.
-"foo".encode("latin1")
+b"foo"
```

Comment:
This useless-looking call to `encode()` is significant - it ensures that the codec is initialized here (where we are presumably executing on the main thread) instead of later in another thread, which has the potential to deadlock.